src/Form/MembershipTermEntityTypeForm.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Drupal\membership\Form;
+
+use Drupal\Core\Entity\EntityForm;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class MembershipTermEntityTypeForm.
+ *
+ * @package Drupal\membership\Form
+ */
+class MembershipTermEntityTypeForm extends EntityForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function form(array $form, FormStateInterface $form_state) {
+    $form = parent::form($form, $form_state);
+
+    $entity_type_manager = \Drupal::service('entity_type.manager');
```

Comment:
Per my notes in the PR, we should be implementing `::create()` and `_construct()` in the dependency injection model for accessing container services. `EntityForm` extends `FormBase`, which implements `ContainerInjectionInterface`.